wpilib/wpilib/adxl362.py

```diff
@@ -179,27 +182,16 @@ def getAccelerations(self):
                 rawData[1] * self.gsPerLSB,
                 rawData[2] * self.gsPerLSB)
 
-    # Live Window code, only does anything if live window is activated.
+    def _updateValues(self):
+        data = self.getAccelerations()
+        self._entryX.setDouble(data[0])
+        self._entryY.setDouble(data[1])
+        self._entryZ.setDouble(data[2])
 
-    def getSmartDashboardType(self):
-        return "3AxisAccelerometer"
-
-    def initTable(self, subtable):
-        if subtable is not None:
-            self.xEntry = subtable.getEntry("X")
-            self.yEntry = subtable.getEntry("Y")
-            self.zEntry = subtable.getEntry("Z")
-            self.updateTable()
-        else:
-            self.xEntry = None
-            self.yEntry = None
-            self.zEntry = None
-
-    def updateTable(self):
-        if self.xEntry is not None:
-            self.xEntry.setDouble(self.getX())
-        if self.yEntry is not None:
-            self.yEntry.setDouble(self.getY())
-        if self.zEntry is not None:
-            self.zEntry.setDouble(self.getZ())
+    def initSendable(self, builder):
+        builder.setSmartDashboardType("3AxisAccelerometer")
+        self._entryX = builder.getEntry("X")
+        self._entryY = builder.getEntry("Y")
+        self._entryZ = builder.getEntry("Z")
 
+        builder.setUpdateTable(self._updateValues)
```

Comment:
Rereading this again, I just realised this is correct. This is probably the simplest way to fix the above two.